site/book/02-concepts/01-packages.md

```diff
@@ -13,13 +13,13 @@ Let's take a look at an example:
 ```shell
 $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/wordpress@v0.3
 $ kpt pkg tree wordpress/
-PKG: wordpress
+Package "wordpress":
 ├── [Kptfile]  Kptfile wordpress
 ├── [service.yaml]  Service wordpress
-├── deployment
+├── "deployment":
```

Comment:
why do we quote the directories and not other things like filenames? It seems inconsistent.

Related: @phanimarupaka did we ever consider alternative notations to presenting packages names? Not suggesting this per se but to 

```
[[wordpress]]
   [Kptfile]
```

Reply:
We can remove quotes for directory names if you want. 

I don't have strong opinion. I proposed few options here and felt this is better. If you have a preference please paste the expected output here and we can go ahead.